Dockerfile

```diff
@@ -24,25 +24,29 @@ FROM base as development
 
 ENTRYPOINT ["poetry","run","flask","run","--host","0.0.0.0"] 
 
-
-# These are the commands to build the image and run docker
-#For Dev instance
-#docker build --target development --tag todo-app:dev .
-
-#without bind mount
-#docker run -p 5000:5000 --env-file .env todo-app
-
-#with bind mount
-#docker run --mount type=bind,src=C:/Users/Rajesh/Devops_project/DevOps-Course-Starter/todo_app,dst=/app/src/todo_app -p 5000:5000 --env-file .env  todo-app:dev
-
-#For Production instance
-#docker build --target production --tag todo-app:prod .
-
-#without bind mount
-#docker run -p 8000:8000 --env-file .env todo-app:prod
-
-#with bind mount
-#docker run --mount type=bind,src=C:/Users/Rajesh/Devops_project/DevOps-Course-Starter/todo_app,dst=/app/src/todo_app -p 8000:8000 --env-file .env  todo-app:prod
-
-
+FROM base as test
+
+RUN poetry install
+RUN apt-get update -qqy && apt-get install -qqy wget gnupg unzip
+# Install Chrome
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub |apt-key add - \
+ && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
+ && apt-get update -qqy \
+ && apt-get -qqy install google-chrome-stable \
+ && rm /etc/apt/sources.list.d/google-chrome.list \
+ && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+# Install Chrome WebDriver
+RUN CHROME_MAJOR_VERSION=$(google-chrome --version | sed -E "s/.* ([0-9]+)(\.[0-9]+){3}.*/\1/") \
+ && CHROME_DRIVER_VERSION=$(wget --no-verbose -O - "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${CHROME_MAJOR_VERSION}") \
+ && echo "Using chromedriver version: "$CHROME_DRIVER_VERSION \
+ && wget --no-verbose -O /tmp/chromedriver_linux64.zip https://chromedriver.storage.googleapis.com/$CHROME_DRIVER_VERSION/chromedriver_linux64.zip \
+ && unzip /tmp/chromedriver_linux64.zip -d /usr/bin \
+ && rm /tmp/chromedriver_linux64.zip \
+ && chmod 755 /usr/bin/chromedriver \
+ && cp /usr/bin/chromedriver .
+
+COPY test ./test
```

Comment:
You need to copy in the .env.test file for your test_trello_mock.py test to work inside the container.

Reply:
Also, you should delete the unused test_trello.py file.

Reply:
test_trelo_mock also works after copying .env.test file 
deleted test_trello.py